### PR TITLE
Adding test to use PV for logHome in MII domain

### PR DESCRIPTION
**Pull request by vanajamukkara**
Jenkins results - 

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1812/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1842/ -- latest
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/1841/ -- latest, failing with known image permission issues same failures as on develop branch

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java [hunk 1]
```diff
@@ -224,27 +266,41 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
...
-          managedServerPrefix + i, domainNamespace);
-      checkPodReady(managedServerPrefix + i, domainUid, domainNamespace);
+      logger.info("Wait for managed server services and pods are created in namespace {0}",
+          domainNamespace);
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
     }
+  }
+
+  /**
+   * Check server logs are written on PV.
```
Comment (anpanigr) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java:282]: We should check for non ZERO size of the log file or look for certain String on the Log file

**Review by anpanigr — APPROVED** [review]
LGTM

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java [hunk 2]
```diff
@@ -758,14 +811,23 @@ private static void createDomainResource(
...
                     .serverPod(new ServerPod()
                             .addEnvItem(new V1EnvVar()
                                     .name("JAVA_OPTIONS")
                                     .value("-Dweblogic.StdoutDebugEnabled=false"))
                             .addEnvItem(new V1EnvVar()
                                     .name("USER_MEM_ARGS")
-                                    .value("-Djava.security.egd=file:/dev/./urandom ")))
+                                    .value("-Djava.security.egd=file:/dev/./urandom "))
+                            .addVolumesItem(new V1Volume()
+                            .name(pvName)
```
Comment (xiancao) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java:825]: add some indentation here to make the code easier to read.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java [hunk 3]
```diff
@@ -758,14 +811,23 @@ private static void createDomainResource(
...
+                            .persistentVolumeClaim(new V1PersistentVolumeClaimVolumeSource()
```
Comment (xiancao) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java:826]: same here

**Review by xiancao — APPROVED** [review]
very minor changes. You can add it to next PR.

**Pull request merged**